docs/ml-clustering.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: global
+title: Clustering - ML
+displayTitle: <a href="ml-guide.html">ML</a> - Clustering
+---
+
+In `spark.ml`, we implement the corresponding pipeline API for 
```

Comment:
How about, "In this section, we introduce the pipeline API for [clustering in mllib](mllib-clustering.html)" ?